### PR TITLE
Revert changes to Poetry_textConcat block

### PR DESCRIPTION
**Pull request by sanchitmalhotra126**
Revert an accidental change to the Poetry_textConcat block which was causing a crash on Poetry in non-English languages.

Block edit page after revert:
Screenshot 2023-11-29 at 12 05 31 PM

Poetry level after revert:

Screenshot 2023-11-29 at 4 45 07 PM

## Warning!!

We have entered Pixel Lock for Hour of Code!

Computer Science Education Week will be happening from Dec 4 - Dec 10. Alongside this event, we will
be launching our new Hour of Code activity. Please consider any risk introduced in this PR that
could affect Dance Lab, instructions, saving and logging student progress, caching, or anything
related to the Hour of Code activities, old or new. Even small changes, such as a different button
color, are considered significant during this time. If this change will affect the new Hour of Code
activity in any way, join the morning change review to get your changes approved prior to merging.
Reach out to the Student Labs team for more details!

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by ebeastlake — APPROVED** [review]
Checked with changes from Nov. 20, and this looks like the correct revert to me: https://github.com/code-dot-org/code-dot-org/commit/48731a48b74646a10469c6de977ecdecaaf2192b

**Review by fisher-alice — APPROVED** [review]
LGTM! Thanks for taking care of this so quickly.

File: i18n/locales/source/dashboard/blocks.yml
Comment (mgc1194) [inline i18n/locales/source/dashboard/blocks.yml:1]: Don't commit changes to this file. When I run the sync this file will be updated. I will go ahead and change this string in crowdin for now.

**Review by jmkulwik — APPROVED** [review]
Looks good to me! Great work everybody!

**Review by cnbrenci — APPROVED** [review]

**Review by cat5inthecradle — APPROVED** [review]

File: i18n/locales/source/dashboard/blocks.yml
Comment (sanchitmalhotra126) [reply i18n/locales/source/dashboard/blocks.yml:1]: Done!

**Review by mgc1194 — APPROVED** [review]

**Comment by sanchitmalhotra126** [comment]
Merging ahead of Drone to deploy today

**Pull request merged**